site/app/templates/LateDaysTablePlugin.twig

```diff
@@ -8,7 +8,7 @@
             {% if diff >= 0 %}
                 <li>{{ late_days_update.since_timestamp|date('m/d/Y') }}: Earned {{ diff }} late day(s)</li>
             {% else %}
-                <li>{{ late_days_ulidate.since_timestamp|date('m/d/Y') }}: Lost {{ -diff }} late day(s)</p>
+                <li>{{ late_days_update.since_timestamp|date('m/d/Y') }}: Lost {{ -diff }} late day(s)</p>
```

Comment:
This change is unrelated to the PR but required to view the grading site until 
https://github.com/Submitty/Submitty/pull/7095

gets merged